amy/settings.py

```diff
@@ -196,6 +201,40 @@
     },
 ]
 
+# GitHub Auth
+AUTHENTICATION_BACKENDS = (
+    'social.backends.github.GithubOAuth2',
+    'django.contrib.auth.backends.ModelBackend',
+)
+SOCIAL_AUTH_ADMIN_USER_SEARCH_FIELDS = ['github']
+SOCIAL_AUTH_GITHUB_KEY = os.environ.get('SOCIAL_AUTH_GITHUB_KEY', '').strip()
+SOCIAL_AUTH_GITHUB_SECRET = os.environ.get('SOCIAL_AUTH_GITHUB_SECRET', '').strip()
+if not SOCIAL_AUTH_GITHUB_KEY or not SOCIAL_AUTH_GITHUB_SECRET:
+    print('Logging using github account will *not* work, '
+          'because you didn\'t set SOCIAL_AUTH_GITHUB_KEY and/or '
+          'SOCIAL_AUTH_GITHUB_SECRET environment variables.')
```

Comment:
`SOCIAL_AUTH_GITHUB_KEY` and `SOCIAL_AUTH_GITHUB_SECRET` should be required if `not DEBUG`, and this message should go to `stderr` because chances are only that output is captured in logs.